src/frontend/utils/conversejs/converse-plugins/chatPlugin.ts

```diff
@@ -1,6 +1,24 @@
+import {
```

Comment:
in the commit message

The chat-plugin message implements a handler **that** uses a callback function
to process received messages.
Depending **on** the type of the message, differents actions are perform**ed**.
...

Reply:
done in rebase, you can check that the description of this commit has changed following this new hash : [](url)